test/rest_api/test_compaction_manager.py

```diff
@@ -15,6 +15,50 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
 
+import sys
+
+from requests import HTTPError
+
+# Use the util.py library from ../cql-pytest:
+sys.path.insert(1, sys.path[0] + '/../cql-pytest')
+from util import new_test_table
+
 def test_compaction_manager_stop_compaction(rest_api):
     resp = rest_api.send("POST", "compaction_manager/stop_compaction", { "type": "COMPACTION" })
+
+def test_compaction_manager_stop_keyspace_compaction(cql, rest_api, test_keyspace):
+    resp = rest_api.send("POST", f"compaction_manager/stop_keyspace_compaction/{test_keyspace}", { "type": "RESHAPE" })
     resp.raise_for_status()
+
+    # non-existing keyspace
+    resp = rest_api.send("POST", f"compaction_manager/stop_keyspace_compaction/{test_keyspace}XXX", { "type": "RESHAPE" })
+    try:
+        resp.raise_for_status()
+    except HTTPError:
+        pass
+
+def test_compaction_manager_stop_keyspace_compaction_tables(cql, rest_api, test_keyspace):
+    with new_test_table(cql, test_keyspace, "a int, PRIMARY KEY (a)") as t0:
+        test_tables = [t0 if not '.' in t0 else t0.split('.')[1]]
```

Comment:
why this if? t0 will always, as returned by new_test_table, contain the keyspace dot table name.

Reply:
until it breaks :)

Reply:
done

Reply:
What was done in 9755? Did you put this comment in the wrong thread?
But never mind, the unnecessary "if" will not hurt anyone. It's ok.